Formula/k/kubevpn.rb

```diff
@@ -1,21 +1,10 @@
 class Kubevpn < Formula
   desc "Offers a Cloud-Native Dev Environment that connects to your K8s cluster network"
   homepage "https://www.kubevpn.cn"
-  url "https://github.com/kubenetworks/kubevpn/archive/refs/tags/v2.2.18.tar.gz"
-  sha256 "bb5c2f19d5dc256794d118a36d9ff2ac7382c38fb7b8ac376e037ea50d99a7c0"
+  url "https://github.com/kubenetworks/kubevpn/archive/refs/tags/v2.2.19.tar.gz"
+  sha256 "4a37db518c9cb79b824addc8852f6a054394c4ca01cea6b4c22722704800bfae"
   license "MIT"
 
-  bottle do
```

Comment:
You can keep the bottle block, it would be managed by the bot

Reply:
ok, got it

Reply:
can you revert the change for that part? thanks!

Reply:
yes, i reverted the change of this part, you can review it now 👍